### ISSUE TITLE
Discussão: padronização das traduções

### ISSUE THREAD

**Issue by marinalimeira** [railsgirls/guides-ptbr#24]
Oie,

Alguns pontos relativos a tradução são importantes a gente padronizar:

- o código (variáveis, nomes de função etc) ficam em inglês ou português?
- muitas vezes aparece o nome `COACH`, deixamos assim ou traduzimos para `TREINADOR(A)`?

**Comment by anaschwendler** [comment]
Na minha opinião: 

> o código (variáveis, nomes de função etc) ficam em inglês ou português?

Talvez a comunicação fique melhor traduzindo, para quem não tem domínio do inglês, se torna mais agradável ler em português :)

> muitas vezes aparece o nome COACH, deixamos assim ou traduzimos para TREINADOR(A)?

Eu gostei muito da tradução para INSTRUTOR que tem sido usado, por mim podemos seguir assim

E levanto mais um ponto:

* Alguns tutoriais tem algumas pequenas falhas, faltam highlights talvez alguns typos, corrigimos na nossa versão, certo?

**Comment by carlosschults** [comment]
No arquivo que estou traduzindo não tem exemplo de código, então não cheguei a ter este problema. Mas concordo com a @anaschwendler , se for para facilitar a compreensão acredito que é melhor traduzir sim.

Quanto a COACH, eu deixei assim mesmo, mas apenas porque o comecinho do arquivo já estava traduzido e estava desta forma, então achei que era um padrão e deixei assim.

**Comment by carlosschults** [comment]
> Alguns tutoriais tem algumas pequenas falhas, faltam highlights talvez alguns typos, corrigimos na nossa versão, certo?

Acho que vale a pena corrigir sim. No meu arquivo, por exemplo, tinha alguns erros, como frases que terminavam abruptamente, e aí não soube o que fazer. Vocês tem sugestões para situações assim?

**Comment by bahbbc** [comment]
>  código (variáveis, nomes de função etc) ficam em inglês ou português?

Concordo com a @anaschwendler, acho que os nomes em português facilitam a compreensão e são menos dificieis de errar ao replicar o código.

> Alguns tutoriais tem algumas pequenas falhas, faltam highlights talvez alguns typos, corrigimos na nossa versão, certo?

Acho que nesse caso vale acrescentar uma linha no README para que também sejam corrigidos problemas de clareza e compreensão do que está escrito, o que acham?

**Comment by anaschwendler** [comment]
@carlosschults 

> Vocês tem sugestões para situações assim?

Eu tenho atentado para corrigir e dar sentido quando reviso o PR, o que sugiro é: faz certinho no PT e abre PR no guide em inglês :)
Já fiz algumas contribuições lá e é bem legal!

**Comment by anaschwendler** [comment]
@bahbbc boa!

> Acho que nesse caso vale acrescentar uma linha no README para que também sejam corrigidos problemas de clareza e compreensão do que está escrito, o que acham?

Total acho isso, dá sentido na tradução e se quiser abre um PR em inglês no original ;)

**Comment by marinalimeira** [comment]
Resumindo o que foi falado até agora:

- Tudo o que não for palavra reservada (e.g. variáveis, nomes de classes), traduzir para o português;

- As referências a `COACH` serão traduzidos como `INSTRUTOR(A)`;

- Se existir algum problema de clareza e compreensão a cerca do que está na versão original, corrigir no momento da tradução (é legal abrir um PR no guia em EN pra corrigir lá também);

A gente deixa essa discussão aberta até finalizarmos as traduções, o que acham? Teremos que revisar todos os arquivos ao final também, porque alguns foram traduzidos antes e não seguiram esse guideline.

**Comment by marinalimeira** [comment]
Dúvida: alguns guias estão desatualizados, como algumas gems não funciona do mesmo jeito da época em que o guia foi feito. Corrigimos agora ou finalizamos a tradução antes? O problema de fazer isso agora é que a gente teria que exigir que o tradutor conheça as ferramentas e atualize isso também. 

Eu prefiro que a gente faça depois. Realizamos o tutorial na nossa máquina e abrimos issues para o que estiver desatualizado.

**Comment by anaschwendler** [comment]
> Corrigimos agora ou finalizamos a tradução antes? 
Idealmente fazemos isso em issues separadas, por exemplo: Update tutorial #XX.
Pode ser antes do original, e depois sugerimos que a pessoa altere no original também.

> Eu prefiro que a gente faça depois. Realizamos o tutorial na nossa máquina e abrimos issues para o que estiver desatualizado

Essa linha mesmo ;)

Acredito que de antemão é melhor que resolvamos as traduções ;)

**Comment by marinalimeira** [comment]
Achei essa nota de tradução no arquivo sobre o SnapCI, acho uma boa nos atentarmos a isso também, apenas por semântica :)
```
Todos os termos que deveriam ser lidos em inglês, recomendo adição do atributo lang="en", conforme especificação de como lidar com internacionalização de documentos HTML. Recomendo a leitura em http://i18n-html-tech-lang.pt.webiwg.org/ (@fititnt, 2016-10-08 23:36)
```
